### PR TITLE
Fix handling of comments and doi at LaTeX check

### PR DESCRIPTION
**Pull request by koppor**
Refs https://github.com/JabRef/jabref/issues/8712#issuecomment-1748970220

@systemoperator I added test cases. Maybe, you could check, too?

### Mandatory checks
- [ ] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [ ] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/main/java/org/jabref/model/entry/field/StandardField.java [hunk 1]
```diff
@@ -63,9 +64,9 @@ public enum StandardField implements Field {
     IDS("ids", FieldProperty.MULTIPLE_ENTRY_LINK),
     INSTITUTION("institution"),
     INTRODUCTION("introduction", FieldProperty.PERSON_NAMES),
-    ISBN("isbn", "ISBN", FieldProperty.ISBN),
-    ISRN("isrn", "ISRN"),
-    ISSN("issn", "ISSN"),
+    ISBN("isbn", "ISBN", FieldProperty.ISBN, FieldProperty.VERBATIM),
+    ISRN("isrn", "ISRN", FieldProperty.VERBATIM),
+    ISSN("issn", "ISSN", FieldProperty.VERBATIM),
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/model/entry/field/StandardField.java:69]: Eprint as well?

Comment (koppor) [reply src/main/java/org/jabref/model/entry/field/StandardField.java:69]: Done. I am not sure if many more should be added. EPRINTTYPE, FOLDER, ... Not sure how we handle that all.

Comment (Siedlerchr) [reply src/main/java/org/jabref/model/entry/field/StandardField.java:69]: we should follow the biber /biblatex defs here

**Review by Siedlerchr — APPROVED** [review]

**Comment by systemoperator** [comment]
> Refs [#8712 (comment)](https://github.com/JabRef/jabref/issues/8712#issuecomment-1748970220)
> 
> @systemoperator I added test cases. Maybe, you could check, too?

Sorry for the delay. Looks good to me and everything works now as expected. :smiley:

**Pull request merged**